db/schema.rb

```diff
@@ -607,7 +607,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.boolean "scanner_working"
-    t.index ["uploader_type", "uploader_id"], name: "index_malware_scan_results_on_uploader_type_and_uploader_id"
+    t.index ["uploader_type", "uploader_id"], name: "index_malware_scan_results_on_uploader"
```

Comment:
I think this migration change has crept in before and should be reverted?

Reply:
Ah yes thanks for spotting, will revert it now!